app/views/shared/_google_tag_manager_body.html.erb

```diff
@@ -0,0 +1,7 @@
+<!-- Google Tag Manager (noscript) -->
+<noscript>
+<iframe src="https://www.googletagmanager.com/ns.html?id=<%= Rails.configuration.tag_manager_id %>"
```

Comment:
I feel like we should wrap both the `body` and `head` partial contents within an if statement similar to how we had it for the `analytics_id`:
`<% if Rails.configuration.respond_to?(:tag_manager_id) -%>`
That way we wouldn't need to add fake ids, and not need to add them for development or test.

Reply:
added if statement & resubmitted.